Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -51,6 +51,24 @@ public override SizeRequest GetDesiredSize(int widthConstraint, int heightConstr
 		{
 			return new SizeRequest(new Size(Context.ToPixels(40), Context.ToPixels(40)));
 		}
+		
+		public virtual void OnMapReady(GoogleMap map)
```

Comment:
Can you please convert this to a public non-virtual method which them calls a protected virtual method?

There are a series of issues with public virtual methods, mostly centering around the fact that you cant sanitize inputs/outputs if issues crop up in the future. By moving to a 2 method solution with a protected virtual we retain the required flexibility.